src/editor/CodeHintList.js

```diff
@@ -306,14 +306,17 @@ define(function (require, exports, module) {
     };
 
     /**
-     * Check whether keyCode is one of the keys that we handle or not.
+     * Check whether Event is one of the keys that we handle or not.
      *
-     * @param {number} keyCode
+     * @param {KeyBoardEvent} keyEvent
      */
-    CodeHintList.prototype.isHandlingKeyCode = function (keyCode) {
+    CodeHintList.prototype.isHandlingEvent = function (event) {
```

Comment:
Changed this to isHandlingEvent so that we could check the actual event object to make sure ctrlKey is called.